XenAdmin/Plugins/Features/TabPageFeature.cs

```diff
@@ -961,6 +961,7 @@ private void writeXML(object context)
                 request.ContentType = "xml";
                 request.ContentLength = Encoding.UTF8.GetBytes(jsCallbackAndData[1]).Length;
                 request.UserAgent = Branding.BRAND_CONSOLE + "\\Plugin";
+                request.Proxy = XenAdminConfigManager.Provider.GetProxyFromSettings(connection, false);

```

Comment:
I think the second parameter in GetProxyFromSettings() needs to be "true", because this is a call to the server (See SessionUrl = connection.Session.Url;)